### PR TITLE
Change kir /ky/ plural settings to same as for English

### LINKED ISSUE #1633: Incorrect plural settings for ky
Currently (2023-06-08) in GlotPress/locales/locales.php, the parameters for Kyrgyz are:

```
		$kir = new GP_Locale();
		$kir->english_name = 'Kyrgyz';
		$kir->native_name = 'Кыргызча';
		$kir->lang_code_iso_639_1 = 'ky';
		$kir->lang_code_iso_639_2 = 'kir';
		$kir->lang_code_iso_639_3 = 'kir';
		$kir->country_code = 'kg';
		$kir->wp_locale = 'kir';
		$kir->slug = 'kir';
		$kir->nplurals = 1;
		$kir->plural_expression = '0';
		$kir->google_code = 'ky';
		$kir->alphabet = 'cyrillic';

```
However, several sources indicate that the plural parameters should be the same as for English.

1. 
http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html

```
ky 	Kyrgyz 	nplurals=2; plural=(n != 1);
```

2.
https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html#ky

```
one => n = 1
"other" includes 0
```

3.
CLDR (per https://php-gettext.github.io/Languages/ )

`n != 1
`

### PR DESCRIPTION
**Pull request by tobifjellner**
Fixes #1633

### CODE REVIEW

**Comment by tobifjellner** [comment]
Are there any possible weird side-effects in various places when the plural parameters suddenly change?

**Comment by pedro-mendonca** [comment]
For the number of plurals, is changing from 1 to 2, probably the strings with plurals that are "current" will keep that status, but will now be incomplete.

Translators and editors should be notified for this changelog, and if possible, the plural strings for the Locale should be marked as fuzzy.

**Review by ocean90 — CHANGES_REQUESTED** [review]
Can you update the test to remove the now obsolete exception for `kir` in https://github.com/GlotPress/GlotPress/blob/0d9920d99adae5513fc8e7a4a22334d7c138e846/tests/phpunit/testcases/test_locales.php#L44

File: tests/phpunit/testcases/test_locales.php [hunk 1]
```diff
@@ -39,10 +39,6 @@ class GP_Test_Locales extends GP_UnitTestCase {
 		// Expected: 0
 		'id',
 
-		// Actual: 0
-		// Expected: n != 1
-		'kir', 'uz',
```
Comment (ocean90) [inline tests/phpunit/testcases/test_locales.php:44]: `uz` needs to stay, only `kir` can be removed.

Comment (tobifjellner) [reply tests/phpunit/testcases/test_locales.php:44]: Thanks.

File: tests/phpunit/testcases/test_locales.php [hunk 2]
```diff
@@ -41,8 +41,8 @@ class GP_Test_Locales extends GP_UnitTestCase {
 
 		// Actual: 0
 		// Expected: n != 1
-		'kir', 'uz',
-
+		'uz',
+
```
Comment (ocean90) [inline tests/phpunit/testcases/test_locales.php]:
```suggestion

```

**Review by ocean90 — DISMISSED** [review]

File: tests/phpunit/testcases/test_locales.php [hunk 3]
```diff
@@ -41,8 +41,7 @@ class GP_Test_Locales extends GP_UnitTestCase {
 
 		// Actual: 0
 		// Expected: n != 1
-		'kir', 'uz',
-
+		'uz',
```
Comment (ocean90) [inline tests/phpunit/testcases/test_locales.php:44]:
```suggestion
		'uz',

```

**Pull request merged**